recipe/meta.yaml

```diff
@@ -16,18 +16,21 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - python >=3.6
-    - make   # [not win]
+    - python
     - m2-make   # [win]
+    - make      # [not win]
     - libtool   # [not win]
+    - libpq 17.0
```

Comment:
This should go in host

Reply:
Thaaaaat it should. Doing too many things at the same time. Good catch.